src/app/functions/server/getExperiments.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { captureException } from "@sentry/node";
+
+/**
+ * Call the Cirrus sidecar, which returns a list of eligible experiments for the current user.
+ *
+ * @see https://github.com/mozilla/experimenter/tree/main/cirrus
+ * @param userId Persistent ID for user, either guest or authenticated
+ * @returns {Promise<any>}
+ */
+export async function getExperiments(
+  userId: string | undefined
+): Promise<unknown> {
+  const serverUrl = process.env.NIMBUS_SIDECAR_URL ?? process.env.SERVER_URL;
```

Comment:
We should possibly confirm w/ SRE that this will work. I think we had an issue w/ some similar logic (although it was wrapped w/ **convict** so maybe not _exactly_ the same) where SRE was setting falsy env vars like `NIMBUS_SIDECAR_URL=""` in some build configs and if we're doing something similar in this project it wouldn't work with the nullish coalescing operator. https://mozilla-hub.atlassian.net/browse/FXA-7621

Looks like this might be the first instance of nullish fallback to another env var, so worth a quick ping perhaps:

```sh
git grep -Pn " \?\? process\.env\." | cat

src/app/functions/server/getExperiments.ts:17:  const serverUrl = process.env.NIMBUS_SIDECAR_URL ?? process.env.SERVER_URL;
```

Although, counterpoint, we seem to do similar things elsewhere already too:

<details>
<summary><kbd>git grep -n " ?? " | grep -E "(process\.env|AppConstants)\."</kbd></summary>

```sh
docs/adr/0005-feature-flags.md:124:  return (process.env.ENABLED_FEATURE_FLAGS ?? '').split(',').includes(flag);
src/app/(nextjs_migration)/(authenticated)/admin/emails/[template]/page.tsx:47:  const admins = process.env.ADMINS?.split(",") ?? [];
src/app/(nextjs_migration)/(authenticated)/admin/emails/page.tsx:17:  const admins = process.env.ADMINS?.split(",") ?? [];
src/app/(nextjs_migration)/(authenticated)/admin/layout.tsx:17:  const admins = process.env.ADMINS?.split(",") ?? [];
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx:51:  const blockList = (process.env.HIBP_BREACH_DOMAIN_BLOCKLIST ?? "").split(",");
src/app/api/utils/auth.ts:224:  const admins = AppConstants.ADMINS?.split(",") ?? [];
src/app/api/v1/premium/route.ts:19:      `${process.env.SERVER_URL ?? ""}/user/breaches`,
src/app/api/v1/user/verify-email/route.ts:16:      (process.env.SERVER_URL ?? "") + "/user/settings",
src/app/functions/server/breachResolution.ts:102:      const blockList = (AppConstants.HIBP_BREACH_DOMAIN_BLOCKLIST ?? "").split(
src/app/functions/server/getExperiments.ts:17:  const serverUrl = process.env.NIMBUS_SIDECAR_URL ?? process.env.SERVER_URL;
src/app/functions/server/isUserSubscribed.ts:28:      process.env.OAUTH_API_URI ?? ""
src/e2e/utils/helpers.js:20:  process.env['E2E_TEST_ENV'] = process.env.E2E_TEST_ENV ?? 'local'
src/e2e/utils/helpers.js:22:  process.env['E2E_TEST_BASE_URL'] = ENV_URLS[process.env.E2E_TEST_ENV] ?? 'https://stage.firefoxmonitor.nonprod.cloudops.mozgcp.net'
src/utils/breachResolution.js:119:      const blockList = (AppConstants.HIBP_BREACH_DOMAIN_BLOCKLIST ?? '').split(',')
src/views/guestLayout.js:16:  const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
src/views/mainLayout.js:16:  const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
src/views/partials/floatingBanner.js:31:    <div tabindex='-1' class='floating-banner' data-type='${AppConstants.FLOATING_BANNER_TYPE ?? ''}' data-delay='${AppConstants.FLOATING_BANNER_DELAY ?? '0'}'  hidden>
```
</details>

---

```js
// env.test.js
const { inspect } = require("node:util");

console.log(`TEST=${inspect(process.env.TEST)}`);
```

```sh
TEST= node env.test.js
```

#### OUTPUT

```text
TEST=''
```

So looks like if SRE doesn't set empty vars it might work, if they set `process.env.NIMBUS_SIDECAR_URL=` then it might fail.